viewer/static/js/app/components/containers/app-container.js

```diff
@@ -147,7 +154,7 @@ class AppContainer extends React.Component {
       populationIds: this.populationIds,
 
       metricIdsToShow: this.metricIdsToShow,
-      allMetricIds: this.allMetricIds,
+      allMetricIdsInDataset: this.allMetricIdsInDataset,
```

Comment:
Does it make sense to drop this in the store state as well? Maybe not yet but I wonder if it would simplify anything down the road.